doc/man/config.rst

```diff
@@ -33,6 +33,11 @@ related values. To find all possible configuration options, run:
     [gentoo]
     push.ask = true
 
+- Sign off all commits for the 'gentoo' repository::
```

Comment:
Let's make it more explanatory, like:

```suggestion
- Add `Signed-off-by` consenting to the `Certificate of Origin`_ to all commits for the 'gentoo' repository::
```

(and link https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin)